Sources/AccountSDKIOSWeb/User/User.swift

```diff
@@ -328,3 +330,23 @@ extension User {
     }
 
 }
+
+// MARK: Shared Keychain
+
+extension User {
+    public func storeSessionInSharedKeychain(appleTeamId: String, completion: @escaping (Result<Void, Error>) -> Void) {
```

Comment:
I am thinking if this should be public, or if the SDK should handle storing to shared keychain internally. 
How is the intended use case for the brands?

Reply:
In my understanding, shared keychain entitlement should be switched on with the correct group to allow saving
 data.

Reply:
So at least we need public method to switch it on?

Reply:
Also i am going to check how we can get teamID programatically